sleap_nn/training/model_trainer.py

```
@@ -813,554 +848,900 @@ def train(self):
+                    skl_name: {
+                        "nodes": skl.nodes,
+                        "edges": skl.edges,
+                        "symmetries": symm,
+                    }
+                }
+
+            # if edges and part names aren't set in config, get it from `sio.Labels` object.
+            head_configs = self.config.model_config.head_configs[self.model_type]
+            for key in head_configs:
+                if "part_names" in head_configs[key][d_num].keys():
+                    if head_configs[key][d_num]["part_names"] is None:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)